weblate/templates/accounts/snippets/login-info.html

```diff
@@ -20,12 +20,15 @@
 
   {% include 'legal/documents/summary.html' %}
 
-{% elif private_commit_email_opt_in %}
-  <p>
-    {% translate "By registering you agree to use your name and e-mail in version control system commits and provide your contribution under license defined by each translated project." %}
-  </p>
 {% else %}
   <p>
-    {% translate "By registering you agree to use your name in version control system commits and provide your contribution under license defined by each translated project." %}
+    {% translate "By registering, you agree that your name and username will be visible to other users signed in to this server as long as your account exists." %}
```

Comment:
@orangesunny We might want to add something like this to https://github.com/WeblateOrg/wllegal/blob/main/wllegal/templates/legal/documents/summary.html as well (this is used on Hosted Weblate and servers we host).

Reply:
Yes!